MUI/src/main/proto/state.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
```

Comment:
We should add a note at the top this file with the permalink to Manticore file that this was copied from